docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -97,11 +91,11 @@ Note that even if you only want to specify the format, you must supply a value i
 
 For typical output files, the date is always written at the beginning of each record. This will consist of the following columns:
 
-year month day hour
+year month day day_seconds
 
 For daily output timestep, "hour" is not written.
```

Comment:
But looks like `hour` is never written anyway. Should this also be `day_seconds`?